.travis.yml

```diff
@@ -38,14 +38,12 @@ before_install:
 
 install:
   - export GIT_FULL_HASH=`git rev-parse HEAD`
-  - conda create -n testenv pip nose pytest python=$TRAVIS_PYTHON_VERSION numpy=$NUMPY scipy scikit-image six coverage cython xraylib lmfit=0.8.3 netcdf4 flake8
+  - conda create -n testenv pip nose pytest python=$TRAVIS_PYTHON_VERSION numpy=$NUMPY scipy scikit-image six coverage cython xraylib lmfit=0.8.3 netcdf4 flake8 pyFAI codecov -c conda-forge
   - source activate testenv
-  - pip install pyFAI
```

Comment:
So does the pip installation
https://travis-ci.org/scikit-beam/scikit-beam/jobs/253358153#L418